packit/cli/types.py

```diff
@@ -66,25 +66,41 @@ def convert(self, value, param, ctx):
             if self.target_branch_param_name:
                 target_branch = self.get_param(self.target_branch_param_name, ctx)
 
+            # General use-case, create fully featured project
+            builder = LocalProjectBuilder()
```

Comment:
Hm, since this is a parameter for click, I am not completely sure if it can be somehow moved elsewhere. Do you have any suggestion? Looking at the use of the class, `convert` will be only used when the command that requires local project is used on the command line so this should hopefully be called only once, if I understand the logic correctly.

Reply:
Ok, I see. It does not make much sense to move/share this instance...